### PR TITLE
Touchups

### PR DESCRIPTION
**Pull request by joepav**
- Button to save name changes
- If a mockup doesn't have a name, it defaults to test #n
- Moved `name` to global tests state
- Ability to delete mockups
- createTestFromIndex validation
- fixing SVG error msgs

### CODE REVIEW

File: src/main/components/Mockup.jsx [hunk 1]
```diff
@@ -15,29 +17,44 @@ const dataTreeOptions = {
...
-  // State
-  const [name, setName] = useState(test.name);
+  const [tests, setTests] = useContext(TestsContext);
 
   // Mode controls test assertions and name edit form—it’s set by child components
   const [mode, setMode] = useState('view');
 
+  function setName(name, testIndex) {
+    const testsClone = [...tests];
+    if (name.replace(/\s/g, '') !== '') testsClone[testIndex].name = name;
```
Comment (joepav) [inline src/main/components/Mockup.jsx:28]: This just makes sure the test name wasn't just whitespace

**Review by Donaldblanc — APPROVED** [review]

File: src/assets/no-data.svg [hunk 2]
```diff
@@ -4,13 +4,13 @@
...
-            <stop stop-color="#808080" stop-opacity="0.1" offset="100%"></stop>
+            <stop stopColor="#808080" stopOpacity="0.25" offset="0%"></stop>
+            <stop stopColor="#808080" stopOpacity="0.12" offset="54%"></stop>
+            <stop stopColor="#808080" stopOpacity="0.1" offset="100%"></stop>
         </linearGradient>
     </defs>
-    <g id="Source" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-        <g id="Data-Source-Empty-State" transform="translate(-223.000000, -235.000000)" fill-rule="nonzero">
+    <g id="Source" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+        <g id="Data-Source-Empty-State" transform="translate(-223.000000, -235.000000)" fillRule="nonzero">
```
Comment (conor-jh) [inline src/assets/no-data.svg:13]: Thanks for fixing all of these 🙌

**Review by conor-jh — DISMISSED** [review]
I think all of this looks good… might need a little walkthrough on the changes to how we’re rendering default names—I got a little lost on the reasoning behind some of the changes. 

One other thing, although I think this can definitely be part of another pull request… it’d be nice to get a little more variety in our button styles so we don’t have quite so many “primary” blue buttons in the mockups panel. 

Maybe a few tertiary styles or icons… would like to discuss and hear your thoughts as well.

**Comment by conor-jh** [comment]
Oh, _one_ other thing before we merge this in—not sure what happened to the changes in `dev` to fix the panel styles. Could they have been overwritten?

**Review by conor-jh — APPROVED** [review]

**Review by conor-jh — APPROVED** [review]

**Pull request merged**